pkgs/development/python-modules/dpcontracts/default.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchFromGitHub
+}:
+
+buildPythonPackage rec {
+  pname = "dpcontracts";
+  # does not have any kind of versioning
+  version = "45cb8542272c2ebe095c6efb97aa9407ddc8bf3c";
```

```suggestion
  version = "unstable-YYYY-MM-DD";
```
You could ask upstream to cut a release.